homeassistant/components/onvif/parsers.py

```diff
@@ -204,7 +205,7 @@ async def async_parse_cell_motion_detector(uid: str, msg) -> Event:
 
         return Event(
             f"{uid}_{msg.Topic._value_1}_{video_source}_{video_analytics}_{rule}",
-            f"{rule} Cell Motion Detection",
+            f"{rule} Motion Detection",
```

Comment:
Is it possible for a camera to send both `CellMotionDetector` and `MotionRegionDetector`? If so, the unique IDs will still be different, but by changing the name to `Motion Detection` how will a user know which is which?

Reply:
Honestly, I dont' know, I thought they were synonyms, but if I'm wrong I could broke something. I think I could revert this change, and instead create 2 different parser, with the same structure but different name. If you agree, I'll proceed